quality/BUILD.bazel

```diff
@@ -59,6 +59,8 @@ licence_test(
         "sw/otbn/crypto/rsa_verify.s",
         "sw/otbn/crypto/handwritten/rsa_verify_3072.s",
         "sw/otbn/crypto/sha512.s",
+        # Code copied unmodified from the SPHINCS+ reference implementation
```

Comment:
nit: 
```suggestion
        # Code derived from the SPHINCS+ reference implementation
```

Reply:
Actually I think we can remove this line now, right?

Reply:
True, the code is now modified. I'll change the header and remove the exemption.